docs_2.0/manual_EN/4.ngql-guide/15.subgraph-and-path/1.get-subgraph.md

```diff
@@ -0,0 +1,78 @@
+
+* Go one step from the vertex with VID "player100" over all types of edges and get the subgraph.
+
+    ```nGQL
+    (root@nebula) [nba]> GET SUBGRAPH 1 STEPS FROM "player100";
+    +--------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+    | _vertices                                                                                                                | _edges                                                                                                                                           |
+    +--------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+    | [(player100) player.name:Tim,player.age:42]                                                                              | [player100-[follow]->player101@0 degree:96,player100-[follow]->player102@0 degree:90,player100-[serve]->team200@0 end_year:2016,start_year:1997] |
+    +--------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+    | [(player101) player.age:36,player.name:Tony Parker,(player102) player.age:33,player.name:LaMarcus Aldridge,(team200) team.name:Warriors] | [player102-[follow]->player101@0 degree:75]                                                                                      |
+    +--------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
+    Got 2 rows (time spent 6289/7423 us)
+    ```
+
+    The returned subgraph is as follows.
+
+![GET SUBGRAPH FROM "player100"](https://docs-cdn.nebula-graph.com.cn/docs_2.0/4.ngql-quide/15.subgraph-and-path/subgraph_1.png)
+
+* Go one step from the vertex with VID "player100" over ingoing `follow` edges and get the subgraph.
+
+    ```nGQL
+    (root@nebula) [nba]> GET SUBGRAPH 1 STEPS FROM "player100" IN follow;
+    +-----------+--------+
+    | _vertices | _edges |
+    +-----------+--------+
+    | []        | []     |
+    +-----------+--------+
+    | []        | []     |
+    +-----------+--------+
+    Got 2 rows (time spent 2292/3091 us)
+    ```
+
+    There is no ingoing `follow` edge to "player100", so no vertex or edge is returned.
+
+* Go one step from the vertex with VID "player100" over outgoing `serve` edges and get the subgraph.
```

Comment:
maybe both follow and serve edges can be applied. the result will be more diverse.